PowerEditor/src/stylers.model.xml

```diff
@@ -342,6 +342,17 @@
             <WordsStyle name="ID" styleID="10" fgColor="0080FF" bgColor="FFFFFF" fontName="" fontStyle="1" fontSize="" />
             <WordsStyle name="IMPORTANT" styleID="11" fgColor="FF0000" bgColor="FFFFFF" fontName="" fontStyle="1" fontSize="" />
             <WordsStyle name="DIRECTIVE" styleID="12" fgColor="0080FF" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="DOUBLESTRING" styleID="13" fgColor="808080" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="SINGLESTRING" styleID="14" fgColor="808080" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="IDENTIFIER2" styleID="15" fgColor="0040E0" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" />

```

Comment:
@donho 
IDENTIFIER2, IDENTIFIER3, EXTENDED_IDENTIFIER, EXTENDED_PSEUDOCLASS, EXTENDED_PSEUDOELEMENT will not work without coresponding list for them:  LIST_2,  LIST_3, LIST_5,  LIST_6,,  LIST_7 setting by `setLexer()` as above.
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/scintilla/lexers/LexCSS.cxx#L75

Could be listed all of them, then it will be easier to add something in the `langs.model.xml` itself, without changing `ScintillaEditView.h`.

Reply:
For CSS2 & CSS3 (IDENTIFIER2 & IDENTIFIER3 respectively) are in current existing list, so I don't think we will add (separate) them in the future. Anyway, I will comment all not used keyword lists for the eventual future added.
Thank you for your review.